src/Sarif.Viewer.VisualStudio/ErrorList/SarifSnapshot.cs

```diff
@@ -133,6 +133,11 @@ public override bool TryGetValue(int index, string columnName, out object conten
                     var error = _errors[index];
                     content = !string.IsNullOrEmpty(error.Message);
                 }
+                else
```

Comment:
I would explicitly state the column here ("`else if (this-is-the-suppressions-column`") for two reasons:
1. in case we add more columns
2. for understandability